libs/wallet-ui/src/components/interactions/flows/transaction-review/transaction-review.tsx

```diff
@@ -116,7 +126,20 @@ export const TransactionReview = ({
         title="Failed to send transaction"
         type={data.error.type}
         message={data.error.error}
-        onClose={onClose}
+        onClose={() => {
+          if (data.transaction) {
+            // this usually happens when there's a traceID mismatch
```

Comment:
question: how do you get a traceID mismatch? that should never happen, as far as i know, unless we have some really serious bug somewhere 🤔

Reply:
It may be the way I'm running it both via app and also via a browser over :34115 🤷🏻 Anyway it didn't let me "finish" the tx so I've found it annoying it was still marked as "in progress".

Reply:
gotcha. yeah, we were not able to get around the issue of parallel apps submitting events to the same backend. if this makes it easier to run things, that's great stuff! i was just worried for a minute that it's something else :)